src/scalarstats.jl

```diff
@@ -45,6 +61,12 @@ function trimmean(x::RealArray, p::Real)
 end
 
 # compute mode, given the range of integer values
+"""
+    mode(a[, rng]) -> Int
```

Comment:
No idea about the docs, but the code clearly returns whatever integer type is used in the input.